### PR TITLE
update k8s documentations

### PR DESCRIPTION
**Pull request by yihuaf**

### CODE REVIEW

File: k8s/kontaind/deployment.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
               mountPath: /var/lib/kubelet/device-plugins
       initContainers:
         - name: kontain-installer
-          image: docker.io/kontainapp/kontain-installer:latest
+          image: docker.io/kontainapp/runenv-kontain-installer:latest
```
Comment (msterin) [inline k8s/kontaind/deployment.yaml:36]: I did not know the images is runenv-* - why ? does it have KM run environment ?

**Review by msterin — COMMENTED** [review]
Doc is good - thanks, but I am not sure about the yaml/image change for kontaind... can you elaborate ?

File: k8s/kontaind/deployment.yaml (hunk 1 above)
Comment (yihuaf) [reply k8s/kontaind/deployment.yaml:36]: Because you asked me to use image.mk to create the image. The naming is controlled by image.mk.

Comment (msterin) [reply k8s/kontaind/deployment.yaml:36]: ah, got it.

**Review by msterin — APPROVED** [review]

**Pull request merged**